docs/guides/support.md

```diff
@@ -218,7 +218,68 @@ missing info.
 info, label the issue as an open proposal, label any relevant metadata, and move
-the issue to the backlog pipeline! 🎉
+the issue to the icebox pipeline! 🎉
+
+The journey of a feature request can be a long and arduous process, depending on
+the ask. Checkout the flow chart to see the path a given feature might take.
+
+```mermaid
+flowchart TD
+    A[Received] --> B[Does it align with the maintainer team's <br />guiding principles for building a design system at IBM?]
+    B -->|No| C[Closed]
+    B -->|Yes| D[Is it supporting a PAL in the DSAG?]
+    D -->|No| E[Is it suporting a PAL not in the DSAG?]
+    E -->|No| F[Is it supporting an IBM product without a PAL?]
+    F -->|No| G[Is it supporting a non-IBM offering that uses Carbon?]
+    G -->|No| C[Closed]
+    D & E & F & G -->|Yes| H([Great! Your request has now been labeled as an `open` proposal. <br />Now we will determine who will be working on this and when.])
+    H --> I[Is this a concept/idea?]
+    H --> J[Is this a fully fleshed out design spec <br /> or code implementation?]
+    I -->|Yes| K[Are you able to dedicate design or <br /> engineering resources to bring this to production?]
+    I <--> |No|J
+    K -->|No| L[Does your request have a low level of effort or a high business impact?]
+    L -->|No| C[Closed]
+    J & K & L -->|Yes| M([Your request has been labeled as an `accepted` <br />proposal and has been added to our Icebox and is <br />being prioritized against competing workstreams])
+    M --> N([Labeled with `Community Contribution` <br /> on GH. Looking for contributions])
+    M --> O([Added to maintainer team's roadmap <br /> and backlog for refinement])
+    N & O --> storm((fa:fa-repeat <br /><br /> Design Crits, Code Reviews,<br /> CAG, DSAG meetings when needed <br /><br />fa:fa-repeat))
+    storm --> ide1
+    subgraph ide1 [Evaluate against a definition of done]
+    P[Storybook]
+    Q[Tests]
+    R[Code]
+    S[Final design spec]
+    T[Website docs]
+    U[Kit tooling]
+    end
+    ide1 -->|Yes| C[Closed]
+
+    style storm fill:#6929c4,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
+    style A fill:#198038,stroke:#fff,stroke-width:2px,color:#fff
+    style C fill:#da1e28,stroke:#000,stroke-width:2px,color:#fff
+    style H fill:#198038,stroke:#fff,stroke-width:2px,color:#fff
+    style M fill:#198038,stroke:#fff,stroke-width:2px,color:#fff
+```
+
+### Definitions
+
+Guiding principless -
+https://carbondesignsystem.com/all-about-carbon/what-is-carbon/#our-guiding-principles
+
+DSAG - The Design System Adoption Guild is an expansive group of designers and
+developers all aligned on the mission of delivering amazing experiences in the
+IBM Products organization. You can learn more about the guild
+[here](https://pages.github.ibm.com/cdai-design/pal/contributing/contribution-framework/intent).
+
+PAL - Pattern Access Library. These are the local systems around IBM that extend
```

Comment:
I think this should be Pattern and Asset Library. 🙃

Reply:
Blargh.. you're totally right!